### PR TITLE
Getting latest fixes onto master

### PR DESCRIPTION
**Pull request by thomasmaurel**

### CODE REVIEW

File: docs/bulk_handover_database.rst [hunk 1]
```diff
@@ -83,7 +84,7 @@ For EG:
   DESCRIPTION="handover new Leopard database"
```
Comment (amonida) [inline docs/bulk_handover_database.rst:84]: This needs changing so it's in line with the non-vert example.

File: docs/bulk_handover_database.rst [hunk 2]
```diff
@@ -37,15 +37,16 @@ To get the list of databases for Fungi:
 .. code-block:: bash
 
   RELEASE=41
-  ./ensembl-production/scripts/process_division.sh EF mysql-eg-pan-prod ensembl_production $RELEASE > eg_handover_databases.txt
+  perl ensembl-metadata/misc_scripts/get_list_databases_for_division.pl $(mysql-ens-meta-prod-1 details script) -division fungi -release $RELEASE > fungi_handover.txt
+
 
 Ensembl:
```
Comment (amonida) [inline docs/bulk_handover_database.rst:43]: Maybe call it 'Vertebrates' instead and replace the EG with 'non-vertebrarates'?

File: docs/bulk_hc_submission.rst [hunk 3]
```diff
@@ -36,16 +36,17 @@ To get the list of databases for Fungi:
 .. code-block:: bash
 
   RELEASE=38
-  ./ensembl-production/scripts/process_division.sh EF mysql-eg-pan-prod ensembl_production $RELEASE > fungi_db_hc.txt
+  perl ensembl-metadata/misc_scripts/get_list_databases_for_division.pl $(mysql-ens-meta-prod-1 details script) -division fungi -release $RELEASE > fungi_db_hc.txt
+
 
 
 Ensembl:
```
Comment (amonida) [inline docs/bulk_hc_submission.rst:43]: Same here, replacing it with 'Vertebrates'.

**Review by amonida — APPROVED** [review]
Minor changes only.

**Review by amonida — APPROVED** [review]

**Comment by thomasmaurel** [comment]
I can't modify this PR easily I am afraid. These changes should be on the release branch too. Closing it

**Pull request closed without merging**